kcidev/subcommands/results/api.py

```diff
@@ -0,0 +1,98 @@
+import urllib
```

Comment:
I'd move this file to `libs/dashboard.py`

Reply:
Good idea. Then we can extend this module with extra functions for new subcommands

Reply:
yes, I agree